vars/filebeat.groovy

```diff
@@ -81,7 +81,7 @@ def stop(Map args = [:]){
   // due to the harcoded creation mask, see https://github.com/elastic/beats/issues/20584
   sh(label: 'Stop filebeat', script: """
     docker exec -t ${config.id} chmod -R ugo+rw /output || echo "Exit code \$?"
-    docker kill ${config.id} || echo "Exit code \$?"
+    docker stop --time 30 ${config.id} || echo "Exit code \$?"
```

Comment:
This will mean we are not removing the container because the worker is ephemeral, right?

Reply:
it is not removed on both cases

Reply:
Good point, it's not `docker container remove`, thanks!